test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/ModelDirectiveTest.cs

```diff
@@ -259,6 +259,13 @@ private DocumentIntermediateNode CreateIRDocument(RazorEngine engine, RazorCodeD
                 }
             }
 
+            // InheritsDirectivePass needs to run before ModelDirective.
+            var pass = new InheritsDirectivePass()
```

Comment:
So even if you remove this code the tests pass? If that's the case we need to update how the tests validate their expectations.